src/components/BasicButton.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './BasicButton.scss';
+
+export default class BasicButton extends Component {
+  render() {
+    const { buttonInfo, buttonName } = this.props;

```

Comment:
```suggestion
    const { info, name } = this.props;

```
사용자는 컴포넌트 이름에서 이미 버튼이라는 사실을 알수 있기 때문에 props의 이름에서 button을 제거해도 전달되는 정보는 동일할 것 같습니다.

Reply:
네!